app/features/worksheet/prompts/worksheet_prompt_multiple_choice_old.txt

```diff
@@ -0,0 +1,8 @@
+You are a subject matter expert on the topic: {topic}.
```

Comment:
If this is an old prompt, you can remove it or consider it as **deprecated**. Legacy components are not considered a part of the main workflow unless you're transitioning from different versions of your code.